product-scenarios/1-manage-users-and-roles/1.1-admin-user-manage-roles/1.1.2-manage-roles-with-scim1/src/test/java/org/wso2/identity/scenarios/test/scim/CreateRoleWithUserSCIM1TestCase.java

```diff
@@ -0,0 +1,127 @@
+
+public class CreateRoleWithUserSCIM1TestCase extends ScenarioTestBase {
+
+    private String username;
+    private String password;
+    private String tenantDomain;
+    private String groupInputFileName;
+    private String userInputFileName;
+    private CloseableHttpClient client;
+    private SCIM1CommonClient scim1Client;
+    private String groupId;
+    private String userId;
+    private String userName;
+
+    @Factory(dataProvider = "manageRolesConfigProvider")
+    public CreateRoleWithUserSCIM1TestCase(String username, String password, String tenantDomain,
+            String userInputFileName, String groupInputFileName) {
+
+        this.username = username;
+        this.password = password;
+        this.tenantDomain = tenantDomain;
+        this.userInputFileName = userInputFileName;
+        this.groupInputFileName = groupInputFileName;
+    }
+
+    @DataProvider(name = "manageRolesConfigProvider")
+    private static Object[][] manageRolesConfigProvider() {
+
+        return new Object[][] {
+                {
+                        ADMIN_USERNAME, ADMIN_PASSWORD, SUPER_TENANT_DOMAIN, "scim1user.json", "scim1group.json"
+                }
+        };
+    }
+
+    @BeforeClass(alwaysRun = true)
+    public void init() throws Exception {
+
+        super.init();
+        scim1Client = new SCIM1CommonClient(getDeploymentProperty(IS_HTTPS_URL));
+        client = HttpClients.createDefault();
+    }
+
+    @Test
+    public void testProvisionUser() throws Exception {
+
+        JSONObject userJSON = scim1Client.getUserJSON(userInputFileName);
+        HttpResponse response = scim1Client.provisionUser(client, userJSON, ADMIN_USERNAME, ADMIN_PASSWORD);
+        assertEquals(response.getStatusLine().getStatusCode(), HttpStatus.SC_CREATED,
+                "User has not been created successfully");
+        JSONObject returnedUserJSON = getJSONFromResponse(response);
+        userId = returnedUserJSON.get(SCIM1Constants.ID_ATTRIBUTE).toString();
+        userName = userJSON.get("userName").toString();
+        assertNotNull(userId, "SCIM1 user id not available in the response.");
+    }
+
+    @Test(dependsOnMethods = "testProvisionUser")
+    public void testCreateGroupWithMembers() throws Exception {
+
+        JSONObject groupJSON = scim1Client.getRoleJSON(groupInputFileName);
+        JSONArray members = new JSONArray();
+            JSONObject member = new JSONObject();
+            member.put(SCIM1Constants.DISPLAY_ATTRIBUTE, userName);
+            member.put(SCIM1Constants.VALUE_PARAM, userId);
+            members.add(member);
+        groupJSON.put(SCIM1Constants.MEMBERS_ATTRIBUTE, members);
+        HttpResponse response = scim1Client.provisionGroup(client, groupJSON, username, password);
+        assertEquals(response.getStatusLine().getStatusCode(), HttpStatus.SC_CREATED,
+                "Group has not been created " + "with members");
```

Comment:
Format the string as "Group has not been created with members"

Reply:
changed